.github/actions/next-stats-action/src/prepare/repo-setup.js

```diff
@@ -154,11 +154,18 @@ module.exports = (actionInfo) => {
                 pkgData.dependencies['@next/swc'] =
                   pkgDatas.get('@next/swc').packedPkgPath
               } else {
-                pkgData.files.push('native')
+                pkgData.files.push('native/*')
               }
             }
           }
 
+          if (pkgData?.scripts?.prepublishOnly) {
+            // There's a bug in `pnpm pack` where it will run
+            // the prepublishOnly script and that will fail.
+            // See https://github.com/pnpm/pnpm/issues/2941
```

Comment:
In a future PR, lets try updating pnpm and removing this hack now that its fixed upstream